R/xportr.R

```diff
@@ -0,0 +1,75 @@
+#' Wrapper to apply all core xportr functions and write xpt
+#'
+#' @param .df A data frame of CDISC standard.
+#' @param var_metadata A data frame containing variable level metadata
+#' @param domain Appropriate CDSIC dataset name, e.g. ADAE, DM. Used to subset
+#'   the metadata object. If none is passed, then name of the dataset passed as
+#'   .df will be used.
+#' @param verbose The action this function takes when an action is taken on the
+#'   dataset or function validation finds an issue. See 'Messaging' section for
+#'   details. Options are 'stop', 'warn', 'message', and 'none'
+#' @param df_metadata A data frame containing dataset level metadata.
+#' @param path Path where transport file will be written. File name sans will be
+#'   used as `xpt` name.
+#' @param strict_checks If TRUE, xpt validation will report errors and not write
+#'   out the dataset. If FALSE, xpt validation will report warnings and continue
+#'   with writing out the dataset. Defaults to FALSE
+#'
+#' @return Returns the input dataframe invisibly
+#' @export
+#'
+#' @examples
+#'
+#' has_pkgs <- requireNamespace("admiral", quietly = TRUE) &&
+#'   requireNamespace("dplyr", quietly = TRUE) &&
+#'   requireNamespace("readxl", quietly = TRUE) &&
+#'   requireNamespace("rlang", quietly = TRUE)
+#'
+#' if (has_pkgs) {
+#'   adsl <- admiral::admiral_adsl
+#'
+#'   spec_path <- system.file(paste0("specs/", "ADaM_admiral_spec.xlsx"), package = "xportr")
```

Comment:
I believe we have data and spec stored in the package now.  You can just call those in this example.  That might fix the failing CMD as well!

Reply:
Good call, thank you! I pulled this example from the README, so I think we should update that at some point too